### PR TITLE
Change File name to avoid conflict

### PR DESCRIPTION
**Pull request by qmjava**
1. rename helpers.rb to moesif_helpers.rb
2. rename Helpers class to MoesifHelpers

### CODE REVIEW

**Comment by xinghengwang** [comment]
we probably need to bump the version and update the gem after, right?

**Comment by praves77** [comment]
Version info is [here](https://github.com/Moesif/moesif-rack/blob/09216c3cf42cdb2163923722f35951c22a045258/moesif_rack.gemspec#L3).

It is used [here](https://github.com/Moesif/moesif-rack/blob/09216c3cf42cdb2163923722f35951c22a045258/Gemfile.lock#L4)

File: lib/moesif_rack/app_config.rb [hunk 1]
```diff
@@ -3,14 +3,14 @@
...
-require_relative './helpers.rb'
+require_relative './moesif_helpers.rb'
 require_relative './regex_config_helper.rb'
 
 class AppConfig
 
     def initialize debug
         @debug = debug
-        @helpers = Helpers.new(debug)
+        @helpers = MoesifHelpers.new(debug)
```
Comment (keyur9) [inline lib/moesif_rack/app_config.rb]: I'd suggest to change the instance variable to `moesif_helpers` as well to avoid any issue.

**Review by keyur9 — APPROVED** [review]
I've left a comment otherwise lgtm

**Review by praves77 — APPROVED** [review]
lftm

**Pull request merged**